### PR TITLE
Add QR code

### PR DESCRIPTION
**Pull request by tadatuta**
WIP!!!

### CODE REVIEW

**Comment by gela-d** [comment]
Just a second, i'll check

File: src/blocks/common.blocks/block-example/block-example.bemtree [hunk 1]
```diff
@@ -74,6 +74,20 @@ block('block-example')(
...
+                        block: 'dropdown',
+                        mods: { switcher : 'link', theme: 'normal' },
+                        mix: [
+                            { block: 'block-example', elem: 'qr' },
+                            { block: 'block-example', elem: 'link' }
+                        ],
+                        switcher: 'QR',
+                        popup: {
+                            block: 'image',
+                            url: 'https://disk.yandex.net/qr/?format=svg&text=' + url,
```
Comment (gela-d) [inline src/blocks/common.blocks/block-example/block-example.bemtree]: in our case, on our site, this url is without domain.
For example: `/_examples/lib/tag/...`
We want here add domain too.

Comment (gela-d) [inline src/blocks/common.blocks/block-example/block-example.bemtree]: In `this` I find only one parent url - in data : reg : langSwitch. 
But I'm not shure that it's right way

Comment (tadatuta) [inline src/blocks/common.blocks/block-example/block-example.bemtree]: yes, that's not good to use `langSwitch`. I'll add fix in a while

**Pull request merged**